integration/basic_test.go

```diff
@@ -26,7 +26,7 @@ func (s *SimpleSuite) TestNoOrInexistentConfigShouldFail(c *check.C) {
 	time.Sleep(500 * time.Millisecond)
 	output := b.Bytes()
 
-	c.Assert(string(output), checker.Contains, "No configuration file found")
+	c.Assert(string(output), checker.Not(checker.Contains), "No configuration file found")
```

Comment:
Hum so it should not print `No configuration file found` anymore. Do we still fail ? with another message ? or not ?
I feel like "if not", those tests are not needed anymore :angel: